app/models/admin/tag_filter.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class Admin::TagFilter
+  KEYS = %i(
+    status
+    name
+    order
+  ).freeze
+
+  attr_reader :params
+
+  def initialize(params)
+    @params = params.to_h.symbolize_keys
+  end
+
+  def results
+    scope = Tag.reorder(nil)
+
+    params.each do |key, value|
+      next if key.to_s == 'page'
```

Comment:
Why the `#to_s`? You did call `#symbolize_keys` on the params hash in the constructor, so keys should already be symbols. Or am I missing something?

Reply:
This was based largely ob the AccountFilter, and I don't know enough about kaminari and our pagination helpers to say with confidence that a change would be okay

Reply:
> This was based largely ob the AccountFilter

Ah, I see. Well, there is the same oddity there. But I see no need to repeat it here :slightly_smiling_face: 

> and I don't know enough about kaminari and our pagination helpers to say with confidence that a change would be okay

I do not believe pagination would be affected here. Pagination happens in the controller and uses the original params, not the ones from this class (that were converted to a hash with symbols as keys).